src/errorpage.cc

```diff
@@ -148,13 +148,12 @@ static void ValidateStaticError(const int page_id, const SBuf &inputLocation);
  \note  hard coded error messages are not appended with %S
  *      automagically to give you more control on the format
  */
-static const struct {
+struct DefaultMessages {
     int type;           /* and page_id */
     const char *text;
-}
-
-error_hard_text[] = {
+};
 
+static std::array<DefaultMessages, 6> HardCodedErrors = {{
```

Comment:
It looks like that HardCodedErrors should also include ERR_REQUEST_PARSE_TIMEOUT and ERR_RELAY_REMOTE. Should we add them in this PR to fix cache.log error messages?
````
2022/10/29 23:09:19.219| 1,3| errorpage.cc(395) loadDefault: WARNING: failed to find or read error text file ERR_RELAY_REMOTE
````

Reply:
If it is easy, let's add them. If others ask us to remove them, we will remove them. Please mention the addition in the PR description.

Reply:
Ok.